lib/ravenx/notification.ex

```diff
@@ -27,7 +27,7 @@ defmodule Ravenx.Notification do
       def dispatch(opts) do
         opts
         |> get_notifications_config
-        |> Enum.map(fn(k, opts) -> {k, Ravenx.Notification.dispatch_notification(opts)} end)
+        |> Enum.map(fn({k, opts}) -> {k, Ravenx.Notification.dispatch_notification(opts)} end)
```

Comment:
Function captures ¿Hot or not?
|> Enum.map(&({&1, Ravenx.Notification.dispatch_notification(&2)}))